entity-framework/core/miscellaneous/logging.md

```diff
@@ -2,14 +2,31 @@
 title: Logging - EF Core
 description: Configuring logging with Entity Framework Core
 author: ajcvickers
-ms.date: 10/27/2016
+ms.date: 10/06/2020
 uid: core/miscellaneous/logging
 ---
 # Logging
 
 > [!TIP]  
 > You can view this article's [sample](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/Miscellaneous/Logging) on GitHub.
 
+## Simple logging
+
+> [!NOTE]
+> This feature was added in EF Core 5.0.
+
+EF Core generates log messages for operations such as executing a query or saving changes to the database. These can be accessed from any application type using [LogTo](/dotnet/api/microsoft.entityframeworkcore.dbcontextoptionsbuilder.logto) when configuring a <xref:Microsoft.EntityFrameworkCore.DbContext> instance. For example:
```

Comment:
Please use xref to refer to API docs.

Reply:
They don't work correctly for referencing methods because the full name of the type and method is used for the link, which makes the sentence virtually unreadable. I want this to literally just say `LogTo`. Replacing this with `Microsoft.EntityFrameworkCore.DbContextOptionsBuilder.LogTo` is not acceptable. Part of the point of the link is to provide a link to additional information so that it does not all have to be given inline with the text.

Reply:
If you use `displayProperty=None` then it will just print name of the method - LogTo.

Reply:
Will try that.

Reply:
...again.